docs/src/pages/[platform]/theming/style-attributes/index.page.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: Component Style Attributes
```

Comment:
Maybe the title should be "Style Props"?

Reply:
+1, we should use the term "props" for React props, and only use "Attributes" when referring to HTML native attributes.

Reply:
+1, agree with Scott's point

Reply:
Good catch, updated